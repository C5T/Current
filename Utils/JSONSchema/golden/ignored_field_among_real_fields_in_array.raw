{"Array":{"element":{"Object":{"field_schema":[["a",{"String":{"values":{"foo":3},"counters":[[3,"foo"]],"instances":3,"nulls":0}}],["b",{"Null":{"occurrences":1}}],["c",{"String":{"values":{"bar":3},"counters":[[3,"bar"]],"instances":3,"nulls":0}}]],"field_index":{"b":1,"c":2,"a":0},"instances":3,"nulls":0}},"instances":1,"nulls":0}}