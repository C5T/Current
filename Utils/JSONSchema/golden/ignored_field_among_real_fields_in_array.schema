// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Element_Object) {
  CURRENT_FIELD(a, std::string);
  // `b` : `null`-s and/or empty arrays, ignored in the schema.
  CURRENT_FIELD(c, std::string);
};

using Schema = std::vector<Schema_Element_Object>;
