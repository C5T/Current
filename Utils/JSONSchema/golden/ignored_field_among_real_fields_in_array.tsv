Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	3	0	3 fields	a, b, c
Schema[].a	String	3	0	1 distinct value	`foo` : 3
Schema[].b	Null	3
Schema[].c	String	3	0	1 distinct value	`bar` : 3
